### ISSUE TITLE
man page errata 

### ISSUE THREAD

**Issue by a2dd** [simsong/tcpflow#205]
Sorry, not enough english to explain correctly, I'll try to reproduce it:

```
:  man tcpflow | col -b | grep scan_http
	      tcpflow -e scan_http -o outdir host sundown
: su -
Password: 
root: tcpflow -e scan_http -o outdir host example.org
Invalid scanner name 'scan_http'
root: tcpflow -e http -o outdir host example.org     
reportfilename: outdir/report.xml
tcpflow: listening on wlp1s0

```
I thought it was a debian (buster) errata but I downloaded the original .tgz file from [](http://downloads.digitalcorpora.org/downloads/tcpflow/ )and:

```
 : pwd
 /tmp/test/tcpflow-1.5.0
 : grep scan_http doc/tcpflow.1.in
 
 \fBtcpflow -e scan_http -o outdir host sundown\fP
```

It seems it comes from the original source and no one noticed.

**Comment by simsong** [comment]
Did you compile from sources?


----
Sent from my phone.

On Feb 7, 2019, at 3:26 PM, a2dd <notifications@github.com> wrote:

Sorry, not enough english to explain correctly, I'll try to reproduce it:

:  man tcpflow | col -b | grep scan_http
	      tcpflow -e scan_http -o outdir host sundown
: su -
Password: 
root: tcpflow -e scan_http -o outdir host example.org
Invalid scanner name 'scan_http'
root: tcpflow -e http -o outdir host example.org     
reportfilename: outdir/report.xml
tcpflow: listening on wlp1s0

I thought it was a debian (buster) errata but I downloaded the original .tgz file from and:

 : pwd
 /tmp/test/tcpflow-1.5.0
 : grep scan_http doc/tcpflow.1.in
 
 \fBtcpflow -e scan_http -o outdir host sundown\fP
It seems it comes from the original source and no one noticed.

**Comment by simsong** [comment]
Fixed in 045035b.
